src/components/footer/_macro-options.md

```diff
@@ -50,3 +51,10 @@
 | Name      | Type                                                        | Required | Description                 |
 | --------- | ----------------------------------------------------------- | -------- | --------------------------- |
 | itemsList | array`<ListItem>` [_(ref)_](/foundations/typography/#lists) | true     | A list of links for the row |
+
+## ExtraPoweredByLogo
+
+| Name      | Type           | Required | Description                                                                             |
+| --------- | -------------- | -------- | --------------------------------------------------------------------------------------- |
+| logoImage | HTML or string | false    | Any HTML to render an image for example embedded `<svg>` or `<img>`.                    |
```

Comment:
"or string" can be removed from this, it doesn't accept a string. Also the description could do with updating the word "Any" can be removed